lib/matplotlib/backends/web_backend/js/nbagg_mpl.js

```diff
@@ -48,7 +48,7 @@ mpl.mpl_figure_comm = function (comm, msg) {
         console.error('Failed to find cell for figure', id, fig);
         return;
     }
-    fig.cell_info[0].output_area.element.one(
+    fig.cell_info[0].output_area.element.on(
```

Comment:
Note, since we are now ignoring some events, we can't just attach on the first trigger (`.one`), but all of them (`.on`)